blockchain/system/rebalance.go

```diff
@@ -108,6 +108,7 @@ func (caller *Kip103ContractCaller) CallContract(ctx context.Context, call kaia.
 	gasPrice := big.NewInt(0) // execute call regardless of the balance of the sender
 	gasLimit := uint64(1e8)   // enough gas limit to execute kip103 contract functions
 	intrinsicGas := uint64(0) // read operation doesn't require intrinsicGas
+	dataTokens := uint64(0)   // read operation has no calldata
```

Comment:
```suggestion
	dataTokens := uint64(0)   // read operation doesn't require intrinsicGas
```
Actually it does have calldata to invoke solidify function.